index.bs

```diff
@@ -78,8 +78,8 @@ Ambient Light Sensor provides information about lighting conditions near
 the device environment. Potential privacy risks include:
 
   - Profiling. Ambient Light Sensor can leak information about user's use
-    patterns and surrounding. This information can be used to enhance user
-    profiling and behavioral analysis.
+    patterns and surrounding cross-origin. This information can be used to
```

Comment:
I defer to a native speaker to give me a grammar lesson.

"Can leak information ... cross-origin."

Sounds good to me.